opacus/layers/dp_lstm.py

```diff
@@ -94,8 +152,9 @@ def __init__(
 
     def forward(
         self,
-        x: torch.Tensor,
+        x: Union[torch.Tensor, Tuple],
         state_init: Tuple[torch.Tensor, torch.Tensor],
+        batch_sizes: Optional[torch.Tensor] = None,
```

Comment:
Couldn't we just read them from x? If x is a PackedSequence, then these will be waiting for us in `x.batch_sizes`: https://pytorch.org/docs/stable/generated/torch.nn.utils.rnn.PackedSequence.html